pkg/datastore/mysql/query.go

```diff
@@ -74,6 +74,8 @@ func buildWhereClause(filters []datastore.ListFilter) string {
 			// Make string of (?,...) which contains the number of `?` equal to the element number of filter.Value
 			valLength := reflect.ValueOf(filter.Value).Len()
 			conds[i] = fmt.Sprintf("%s %s (?%s)", filter.Field, filter.Operator, strings.Repeat(",?", valLength-1))
+		case "MEMBER OF":
+			conds[i] = fmt.Sprintf("? %s (%s)", filter.Operator, filter.Field)
```

Comment:
Please add test for this in [query_test.go/TestBuildFindQuery](https://github.com/pipe-cd/pipe/blob/master/pkg/datastore/mysql/query_test.go#L386) 😁

Reply:
Right right